style/main.css

```diff
@@ -44,13 +90,753 @@
+    text-align: left;
+    font-size: 42px;
+    line-height: 60px;
+  }
+  .hero-section {
+    padding-top: 70px;
+    padding-bottom: 76px;
+  }
+}
+@media screen and (min-width: 992px) {
+  .main-title {
+    margin-top: 30px;
+    font-size: 50px;
+    line-height: 65px;
+  }
+  .main-text {
+    text-align: left;
+    padding-top: 50px; 
+    padding-bottom: 50px;
+  }
+}
+@media screen and (min-width: 1280px) {
+  .image-thanks {
+    padding-left: 10px;
+    min-width: 345px;
+    padding-right: 50px;
+    min-height: 300px;
+  }
+  .padding-top {
+    padding-top: 55px;
+  }
+  .main-title {
+    margin-top: 0;
+    font-size: 56px;
+    line-height: 69px;
+  }
+  .main-text {
+    margin: 0;
+    text-align: left;
+    padding-top: 36px; 
+    padding-bottom: 52px;
+  }
+}
+.main-title {
+  font-style: medium;
+}
+.how-we-work {
+  padding-top: 20px;
+  background: #f1f1f2;
+  padding-bottom: 20px;
+}
+.end-section {
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+.text-title {
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 35px;
+  font-style: normal;
+  text-align: center;
+}
+@media screen and (min-width: 576px) {
+  .end-section {
+    padding-top: 40px;
+    padding-bottom: 40px;
+  }
+  .how-we-work {
+    padding-top: 40px;
+    padding-bottom: 60px;
+  }
+  .text-title {
+    font-size: 25px;
+    line-height: 40px;
+  }
+}
+@media screen and (min-width: 768px) {
+  .how-we-work {
+    padding-top: 70px;
+    padding-bottom: 100px;
+  }
+  .end-section {
+    padding-top: 80px;
+    padding-bottom: 80px;
+  }
+  .text-title {
+    font-size: 30px;
+    line-height: 40px;
+    margin: 0;
+    text-align: left;
+  }
+  .col-element {
+    max-width: 430px;
+  }
+  .work-flex {
+    justify-content: space-between;
+  }
+  .how-we-work {
+    justify-content: space-between;
+  }
+  .text-custom {
+    text-align: justify;
+  }
+  .hero-col {
+    justify-content: space-between;
+    width: 50%;
+  }
+}
+@media screen and (min-width: 1280px) {
+  .hero-col {
+    padding-left: 10px;
+  }
+  .text-title {
+    font-size: 32px;
+    line-height: 45px;
+  }
+}
+.gift {
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+@media screen and (min-width: 576px) {
+  .gift {
+    padding-top: 40px;
+    padding-bottom: 40px;
+  }
+}
+.holidays {
+  padding-top: 50px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+.holidays-block {
+  display: flex;
+  flex-direction: column;
   align-items: center;
+  background: #fafafa;
+  width: 270px;
+  margin: 0 auto;
+  margin-bottom: 20px;
+}
+@media screen and (min-width: 992px) {
+  .holidays-block {
+    align-items: flex-start;
+  }
+  .text-custom {
+    padding-top: 30px;
+    padding-bottom: 30px;
+  }
+}
+.holidays-image {
+  max-width: 270px;
+  max-height: 270px;
+}
+.holidays-text {
+  margin: 0;
+  padding-top: 25px;
+  padding-bottom: 75px;
+}
+.holidays-link {
+  padding-bottom: 25px;
+  color: #000;
+  transition: 0.4s;
```

Comment:
Where did you lose transition-property? By default it is 'all', but this is not exactly we need.